opentech/apply/funds/admin.py

```diff
@@ -74,9 +34,34 @@ class LabAdmin(ModelAdmin):
     menu_label = 'Labs'
 
 
+class NoDeletePermission(PermissionHelper):
```

Comment:
Will this be useful for other apps/models?

Reply:
Perhaps. But only here for now.